src/shared/functions/index.ts

```diff
@@ -40,3 +40,44 @@ export function sanitizeObject<T>(
   }
   return params.object;
 }
+
+/**
+ * Returns new array with element(s) from value param or undefined if value was not defined
```

Comment:
It looks like line 53 returns null explicitly, instead of undefined

```suggestion
 * Returns new array with element(s) from value param or null if value was not defined
```